<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStock</class>
 <widget class="QDialog" name="AddStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Code: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Code"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Categorie:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_categorie"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Designation: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Designation"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Quantite: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_Quantite">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QRadioButton" name="radioButton_prix_achat_par_packet">
             <property name="text">
              <string>Prix Achat par Packet</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_prix_achat_par_montant_total">
             <property name="text">
              <string>Prix Achat par montant total</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Prix:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_prix_achat">
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Prix Vente</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Gros:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_PV_Gros">
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Demi Gros:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_PV_demi_Gros">
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Detail:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_PV_Detail">
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Date d'achat:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_date_achat">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Date d'expiration:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_date_expiration">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_view">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
